OWML.Launcher/App.cs

```diff
@@ -159,12 +167,17 @@ private void PatchGame(IList<IModData> mods)
             }
         }
 
-        private void StartGame(string[] args)
+        private void StartGame(string[] args, bool enableVR)
         {
             _writer.WriteLine("Starting game...");
             try
             {
-                Process.Start($"{_owmlConfig.GamePath}/OuterWilds.exe", string.Join(" ", args));
+                var gameArgs = string.Join(" ", args);
+                if (enableVR)
+                {
+                    gameArgs += VrArgument;
```

Comment:
shouldn't there be a space in front in some cases? 
if so, how about adding the arg to the list before joining?

Reply:
It's always the last argument, so no? I can do it via the array anyway. I started off that way but it looked messier to add stuff to the array than to the string 😅

Reply:
ah, the vr arg has a space in front, so it doesn't matter 🤷